src/standalone/Vegetation/Canopy.jl

```diff
@@ -692,10 +693,37 @@ function CanopyModel{FT}(;
         @assert typeof(photosynthesis) <: PModel{FT} "When using PModelConductance for stomatal conductance, you must also use PModel for photosynthesis"
     end
 
+    # Height consistency check between biomass and hydraulics:
```

Comment:
Could you move this comment into the docstring? That way it'll be visible in the docs too